bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       # by default, Deployments spin up the new pod before terminating the old one
       # but we don't want that - because ovsdb holds the lock.
       maxSurge: 0
-      maxUnavailable: 1
+      maxUnavailable: 33%
```

Comment:
could this be 10%? I thought I remember reading that k8s always rounds up, so if we specify 10% it should still be 1/3. If for some reason we scaled to 7 masters, it will still be safer than doing 33%. Thoughts?

Reply:
@trozet it could be, but the tests check for 10% *or* 33%, so I don't see why this hsouldn't work. The CI failure message was:

```
Mar 15 09:37:25.922: FAIL: Daemonsets found that do not meet platform requirements for update strategy:
  expected daemonset openshift-ovn-kubernetes/ovnkube-master to have maxUnavailable 10% or 33% (see comment) instead of 1
```

Reply:
Control plane pods should probably still be maxUnavailable 1.  I will add something to the test that calls that out - there is really no harm updating masters 1 by 1 and that is intentional

Reply:
We will never support 10 masters so leaving this as 1 is probably clearer than 10%

Reply:
@smarterclayton can you link the PR you're doing to allow maxUnavailable=1 for control-plane pods when you post it?

Reply:
currently all our pods fall afoul of your test:

```
fail [github.com/onsi/ginkgo@v4.7.0-origin.0+incompatible/internal/leafnodes/runner.go:113]: Mar 14 15:24:03.940: Daemonsets found that do not meet platform requirements for update strategy:
  expected daemonset openshift-ovn-kubernetes/ovnkube-master to have maxUnavailable 10% or 33% (see comment) instead of 1
  expected daemonset openshift-ovn-kubernetes/ovs-node to have maxUnavailable 10% or 33% (see comment) instead of 1
```